extensions/wikia/AdEngine/js/context/slotsContext.js

```diff
@@ -0,0 +1,65 @@
+/*global define*/
+define('ext.wikia.adEngine.context.slotsContext', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.utils.adLogicZoneParams',
+	'wikia.geo',
+	'wikia.instantGlobals',
+	'wikia.log'
+], function (adContext, params, geo, instantGlobals, log) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.context.slotsContext',
+		slots = null;
```

Comment:
this can be initiated as an empty object, remove the `slots = {}` from setup slots